plugins/ocm/README.md

```diff
@@ -1,104 +1,114 @@
 # Open Cluster Management plugin for Backstage
 
-This plugin integrates your Backstage instance with Open Cluster Management's MultiClusterHub and MultiCluster Engine.
+The Open Cluster Management (OCM) plugin integrates your Backstage instance with the `MultiClusterHub` and `MultiCluster` engines of Open Cluster Management.
 
 ## Prerequisites
 
-1. OCM deployed and configured on a kubernetes cluster
-2. Installed [Kubernetes plugin for Backstage](https://backstage.io/docs/features/kubernetes/overview)
-3. Kubernetes plugin is properly configured and able to connect to the Hub cluster for OCM via a `ServiceAccount` (the cluster is accessed by backstage backend, therefore it requires a permanent connection with constant RBAC scope)
-4. Following `ClusterRole` must be granted to `ServiceAccount` accessing the Hub cluster:
-   ```yaml
-   kind: ClusterRole
-   apiVersion: rbac.authorization.k8s.io/v1
-   metadata:
-     name: backstage-ocm-plugin
-   rules:
-     - apiGroups:
-         - cluster.open-cluster-management.io
-       resources:
-         - managedclusters
-       verbs:
-         - get
-         - watch
-         - list
-     - apiGroups:
-         - internal.open-cluster-management.io
-       resources:
-         - managedclusterinfos
-       verbs:
-         - get
-         - watch
-         - list
-   ```
+- OCM is deployed and configured on a Kubernetes cluster.
+- [Kubernetes plugin for Backstage](https://backstage.io/docs/features/kubernetes/overview) is installed.
+- Kubernetes plugin is configured and able to connect to the Hub cluster for OCM using a `ServiceAccount`. The cluster is accessed using Backstage backend, therefore, the cluster requires a permanent connection with constant RBAC scope.
+- The `ClusterRole` must be granted to `ServiceAccount` accessing the Hub cluster as follows:
+  ```yaml
+  kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: backstage-ocm-plugin
+  rules:
+    - apiGroups:
+        - cluster.open-cluster-management.io
+      resources:
+        - managedclusters
+      verbs:
+        - get
+        - watch
+        - list
+    - apiGroups:
+        - internal.open-cluster-management.io
+      resources:
+        - managedclusterinfos
+      verbs:
+        - get
+        - watch
+        - list
+  ```
 
 ## Capabilities
```

Comment:
I would move information about capabilities to before the prerequisites (abstract / intro)

Reply:
Good point. Thank you. 
Moved it before Prerequisites.